go-sdk/pkg/bundles/shared/interface.go

```diff
@@ -0,0 +1,31 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Again a nit, but could we call this `handshake.go`?

Reply:
I guess we can yeah -- there was more things in here, but I moved them elsewhere. Good idea.